cargo-shuttle/src/lib.rs

```diff
@@ -1792,12 +1792,21 @@ impl Shuttle {
 
             let metadata = async_cargo_metadata(manifest_path.as_path()).await?;
             let packages = find_shuttle_packages(&metadata)?;
-            let package_name = packages
+            // TODO: support overriding this
+            let package = packages
                 .first()
-                .expect("at least one shuttle crate in the workspace")
-                .name
-                .to_owned();
+                .expect("at least one shuttle crate in the workspace");
+            let package_name = package.name.to_owned();
             deployment_req_beta.package_name = package_name;
+
+            // TODO: add these to the request and builder
+            let (_no_default_features, _features) = if package.features.contains_key("shuttle") {
+                (true, vec!["shuttle".to_owned()])
+            } else {
+                (false, vec![])
+            };
```

Comment:
Shouldn't we address this todo?

Reply:
Leaving it there for the next update to beta's deploy/build flow.